### DISCUSSION TITLE
chia network wallet's update

### DISCUSSION THREAD

**Discussion by mmrezaak** [Chia-Network/chia-blockchain#16785] [Support]
Hello

I had a question

I wanted to know when the new version of Chia Wallet will be uploaded for November

Because there is a clawing problem in most wallets, and I think you have solved this problem in the new version, I would be grateful if you could inform me of the exact time of the new wallet update.

thank you

**Comment by BrandtH22** [comment]
Hey @mmrezaak , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia). One of the major clawback bugs was fixed with the 2.1 release and another less common one is still being investigated for a solution.

Even with the other bug there is now a method to force pending clawbacks to go through by running the following commands:

1. Get the clawback transaction id by running `chia wallet get_transactions --clawback`
2. Force the claim by running:
`chia rpc wallet spend_clawback_coins '{"coin_ids":["TRANSACTION_ID"], "force":true, "fee":0}'`

Note - If on windows you will need to escape the quotes for the clawback claim command:
`chia rpc wallet spend_clawback_coins '{\"coin_ids\":[\"TRANSACTION_ID\"], \"force\":true, \"fee\":0}'`

Note 2 - you will need chia version 2.1 or greater to run these commands and can download the latest version from https://www.chia.net/downloads/

**Comment by mmrezaak** [comment]
We used this command code before but it didn't work, if you have any other
solution please let me know.

thank you

**Reply by BrandtH22** [reply]
Hey @mmrezaak , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia). 

That command is still fully functional for forcing clawbacks. Can you confirm:
- What is the transaction id for the clawback?
- Does the clawback appear as pending after a full resync? (renaming the wallet db and starting chia to let the wallet sync from scratch)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
